ewkcoffea/modules/selection_Run3_2Lep.py

```diff
@@ -16,150 +11,114 @@
     "2022" : {
         "EGamma" : [
             "Ele23_Ele12_CaloIdL_TrackIdL_IsoVL_DZ",
+            "Ele23_Ele12_CaloIdL_TrackIdL_IsoVL",
+            "Ele30_WPTight_Gsf",
+            "Ele32_WPTight_Gsf",
+            "Ele32_WPTight_Gsf_L1DoubleEG",
+            "Ele35_WPTight_Gsf",
+            "Ele115_CaloIdVT_GsfTrkIdT",
+            "DoubleEle25_CaloIdL_MW",
+            "DoublePhoton70",
+        ],
+        "SingleMuon" : [
+            "IsoMu24",
+            "IsoMu27",
+            "Mu50",
+        ],
+        "DoubleMuon" : [
+            "Mu17_TrkIsoVVL_Mu8_TrkIsoVVL_DZ_Mass3p8",
+            "Mu17_TrkIsoVVL_Mu8_TrkIsoVVL_DZ_Mass8",
         ],
         "Muon" : [
             "Mu17_TrkIsoVVL_Mu8_TrkIsoVVL_DZ_Mass3p8",
+            "Mu17_TrkIsoVVL_Mu8_TrkIsoVVL_DZ_Mass8",
+            "IsoMu24",
+            "IsoMu27",
+            "Mu50",
         ],
         "MuonEG" : [
+            "Mu12_TrkIsoVVL_Ele23_CaloIdL_TrackIdL_IsoVL_DZ",
+            "Mu23_TrkIsoVVL_Ele12_CaloIdL_TrackIdL_IsoVL",
             "Mu23_TrkIsoVVL_Ele12_CaloIdL_TrackIdL_IsoVL_DZ",
             "Mu8_TrkIsoVVL_Ele23_CaloIdL_TrackIdL_IsoVL_DZ",
         ]
     }
 
 }
 
-trgs_for_matching = {
-
-    "2022" : {
-        "m_m" : {
-            "trg_lst" : ["Mu17_TrkIsoVVL_Mu8_TrkIsoVVL_DZ_Mass3p8"],
-            "offline_thresholds" : [10.0,10.0],
-        },
-        "e_e" : {
-            "trg_lst" : ["Ele23_Ele12_CaloIdL_TrackIdL_IsoVL_DZ"],
-            "offline_thresholds" : [10.0,10.0],
-        },
-        "m_e" : {
-            "trg_lst" : ["Mu23_TrkIsoVVL_Ele12_CaloIdL_TrackIdL_IsoVL_DZ"],
-            "offline_thresholds" : [10.0,10.0],
-        },
-        "e_m" : {
-            "trg_lst" : ["Mu8_TrkIsoVVL_Ele23_CaloIdL_TrackIdL_IsoVL_DZ"],
-            "offline_thresholds" : [10.0,10.0],
-        },
-    }
-}
-
 
 # Hard coded dictionary for figuring out overlap...
 #   - No unique way to do this
 #   - Note: In order for this to work properly, you should be processing all of the datastes to be used in the analysis
 #   - Otherwise, you may be removing events that show up in other datasets you're not using
+
+# Note: The reason for era dependency in 2022 is the change from having a SingleMuon and DoubleMuon dataset to just a Muon dataset in era C
```

Comment:
For future reference, I think it would be good to add to this comment to mention the run number where this happens.